### PR TITLE
Add support for Cisco UCS Service Profile Discovery

### PR DESCRIPTION
**Pull request by uppalk1**
depends on RackHD/on-tasks/pull/420

### CODE REVIEW

**Comment by uppalk1** [comment]
@tannoa2 @geoff-reid @keedya

File: lib/graphs/ucs-service-profile-discovery-graph.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
+// Copyright 2017, EMC, Inc.
```
Comment (anhou) [inline lib/graphs/ucs-service-profile-discovery-graph.js]: please use the newest copyright declaration

File: spec/lib/graphs/ucs-service-profile-discovery-graph-spec.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+// Copyright 2017, EMC, Inc.
```
Comment (anhou) [inline spec/lib/graphs/ucs-service-profile-discovery-graph-spec.js]: please use the newest copyright declaration

**Comment by geoff-reid** [comment]
test this please

**Pull request merged**